<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Menu</class>
 <widget class="QWidget" name="Main_Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main_Menu</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>600</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">#page{border-image:url(:/new/prefix1/res/BackGround2.png)}</string>
    </property>
    <widget class="QPushButton" name="Button_author">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_local_game">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string>本地游戏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_net_game">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string>网络游戏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_exit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>440</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string>退出游戏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_set">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_IP">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>81.69.223.92</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_PT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>4396</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_save_set">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string>save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_initialization">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string>官匹</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ip">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_information">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>输入IP地址和端口号后点击保存进行网络游戏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_update_diary">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">#page_2{border-image:url(:/new/prefix1/res/BackGround2.png)}</string>
    </property>
    <widget class="QLabel" name="label_create_name">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>474</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>房间标题：</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>11</y>
       <width>550</width>
       <height>400</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>400</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_refresh">
     <property name="geometry">
      <rect>
       <x>428</x>
       <y>418</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string>刷新战局</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LineEdit1">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>474</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>如果你足够自信就来挑战我吧！！！</string>
     </property>
     <property name="maxLength">
      <number>64</number>
     </property>
     <property name="cursorPosition">
      <number>16</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_reconnect">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>530</y>
       <width>400</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string>连接服务器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_create">
     <property name="geometry">
      <rect>
       <x>428</x>
       <y>474</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string>创建房间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_exit_net">
     <property name="geometry">
      <rect>
       <x>428</x>
       <y>530</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround1.png)</string>
     </property>
     <property name="text">
      <string>退出网络对战</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_connect_stat">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>418</y>
       <width>400</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_people">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>580</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
